rules/library/xcconfig.bzl

```diff
@@ -134,6 +162,20 @@ def settings_from_xcconfig(xcconfig):
     ibtool_copts = []
     linkopts = []
 
+    xcconfig = dicts.add(
+        {
+            "CLANG_ENABLE_MODULE_DEBUGGING": "NO",
```

Comment:
Why are these two special cased here? Seems potentially a bit buried. Would it be worth pulling these out to somewhere more visible?

Reply:
can pull into a constant and comment, but we explicitly disable these because they turn on flags we really don't want when `CLANG_ENABLE_MODULES = YES`

Reply:
Should this be up to the caller to pass this in?

Reply:
IMO no -- it would mean that every build would have warnings (from the second setting), and would be non-reproducible (from the first)